mission_control/navigator_launch/launch/master_boat.launch

```diff
@@ -1,3 +1,5 @@
 <launch>
```

Comment:
Master in terms of launching everything - like this is the master launch file for all sub-components